cmd/crane/cmd/mutate.go

```diff
@@ -78,6 +87,14 @@ func NewCmdMutate(options *[]crane.Option) *cobra.Command {
 				return err
 			}
 
+			// set envvars if specified - split to a function as it is more
+			// complex then labels because in the config the key value pairs
+			// are just strings with a = in them
```

Comment:
Thanks for this context, and I agree, but I'm not sure the comment is useful outside the context of this code review.
```suggestion
			// set envvars if specified.
```